@xen-orchestra/lite/src/components/RouterTab.vue

```diff
@@ -1,26 +1,17 @@
 <template>
-  <RouterLink v-slot="{ isActive, href }" :to="isDisabled ? '' : to" custom>
-    <UiTab :active="isActive" :disabled :href tag="a">
+  <RouterLink v-slot="{ isActive, href }" :to custom>
+    <TabItem :active="isActive" :disabled :href="disabled ? undefined : href" :tag="disabled ? 'div' : 'a'">
```

Comment:
Let's use the default tag

```suggestion
    <TabItem :active="isActive" :disabled :href="disabled ? undefined : href" :tag="disabled ? undefined : 'a'">
```